app/src/main/java/com/dd3boh/outertune/playback/MusicService.kt

```diff
@@ -434,7 +435,9 @@ class MusicService : MediaLibraryService(),
 
     private suspend fun recoverSong(mediaId: String, playerResponse: PlayerResponse? = null) {
         var playbackUrl = database.format(mediaId).first()?.playbackUrl
-
+        if (playbackUrl != null) {
+            Log.i("playbackurl", playbackUrl)
+        }
```

Comment:
no loging. If you want to have debug logging, hid it behind a debugging flag. for example: https://github.com/DD3Boh/OuterTune/blob/e82a47ead8393a21eeb95626f610432affcbb0fc/app/src/main/java/com/dd3boh/outertune/ui/utils/LocalMediaUtils.kt#L20